0000-catalog.md

```diff
@@ -541,181 +574,228 @@ Address of the Catalog to be deleted
+  is invalid.
+- The signer of the transaction must be an agent in the Pike state and must
+  belong to an
+  organization in Pike state, otherwise the transaction is invalid.
+  The owner in the product must match the organization that the agent belongs to,
+  otherwise the transaction is invalid.
+- The agent must have the permission can_update_product for the organization,
+  otherwise
+  the transaction is invalid.
+- The properties must be valid for the catalog_product schema; its properties
+  must only contain properties that are included in the catalog_product Schema.
 
 The inputs for CatalogProductUpdateAction must include:
 
 - Address of the Agent submitting the transaction
 - Address of the Organization the Product is being updated for
 - Address of the Product to be updated
 
-The outputs for ProductUpdateAction must include:
+The outputs for CatalogProductUpdateAction must include:
 
 - Address of the updated catalog_product
 
 ### CatalogProductDeleteAction
-CatalogProductDeleteAction removes an existing product from state. The transaction should be 
-submitted by an agent, identified by its signing key, acting on behalf of the organization 
-that corresponds to the org_id in the product being updated. (Organizations and agents are 
+
+CatalogProductDeleteAction removes an existing product from state. The
+transaction should be
+submitted by an agent, identified by its signing key, acting on behalf of the
+organization
+that corresponds to the org_id in the product being deleted. (Organizations and
+agents are
 defined by the Pike smart contract.)
+
 ```
-message CatalogProductDeleteAction { 
+message CatalogProductDeleteAction {
     // catalog_id and product_id are used in deriving the state address
-    string catalog_id = 1; 
-    string product_id = 2; 
-} 
+    string catalog_id = 1;
+    string product_id = 2;
+}
 ```
-If the Grid setting grid.product.allow_delete is set to false, this transaction is invalid. (sys admin setting)
-The default value for grid.product.allow_delete is true. This setting is stored using the 
+
+If the Grid setting grid.product.allow_delete is set to false, this transaction
+is invalid. (sys admin setting)
+The default value for grid.product.allow_delete is true. This setting is stored
+using the
 Sawtooth Settings smart contract, more information can be found here.
 
 Validation requirements:
 
-- If a catalog_product with product_id does not exist the transaction is invalid.
-- The signer of the transaction must be an agent in the Pike state and must belong to an 
-organization in Pike state, otherwise the transaction is invalid.
-- The owner in the product must match the organization that the agent belongs to, otherwise 
-the transaction is invalid.
-- The agent must have the permission “can_delete_product” for the organization otherwise the 
-transaction is invalid.
+- If a catalog_product with product_id does not exist the transaction is
```

Comment:
everywhere you say product_id meaning the id for the catalog product it should say catalog_product_id

Reply:
✅